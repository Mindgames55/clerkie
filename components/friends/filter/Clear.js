import styles from './Clear.module.css'

export default function Clear() {
    return (
        <button className={styles.clear}>Clear all</button>
    )
}