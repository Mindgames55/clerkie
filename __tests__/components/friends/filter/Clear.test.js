import { render, screen } from '@testing-library/react';
import Clear from '@/components/friends/filter/Clear'

it('renders Clear button correctly', () => {
    const { container } = render(<Clear />);
    expect(container).toMatchSnapshot();
});