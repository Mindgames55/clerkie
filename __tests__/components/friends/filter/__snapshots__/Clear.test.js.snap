// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Clear button correctly 1`] = `
<div>
  <button
    class="clear"
  >
    Clear all
  </button>
</div>
`;
